test/acl_command_queue_test.cpp

```diff
@@ -173,6 +173,7 @@ MT_TEST(acl_command_queue, create) {
   // check if acl_command_queue_is_valid is working properly
   {
     cl_command_queue fake_cq = acl_alloc_cl_command_queue();
+    CHECK(fake_cq != 0);
```

Comment:
Shall I change this to `CHECK(fake_cq != NULL);`?

Reply:
How about just ```CHECK(fake_cq);```?

Reply:
I see, I'll try that.